src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -78,6 +124,38 @@ public Game fromString(String string) {
       }
     });
 
+    length.getItems().addAll(PathUnits.LENGTHS);
+    length.getSelectionModel().selectFirst();
+    length.setConverter(new StringConverter<>() {
+      @Override
+      public String toString(Unit<Length> object) {
+        return object.getName();
+      }
+
+      @Override
+      public Unit<Length> fromString(String string) {
+        throw new UnsupportedOperationException();
+      }
+    });
+
+    var lengthUnit = EasyBind.monadic(length.getSelectionModel().selectedItemProperty());
+    timeControls.forEach(control -> control.tooltipProperty().setValue(new Tooltip("Time delta between points")));
```

Comment:
Use `control.setTooltip()` instead of directly calling the property's `setValue()` method

Also applies to the lines below this